src/test/java/com/wavefront/WavefrontNozzleApplicationTests.java

```diff
@@ -0,0 +1,17 @@
+package com.wavefront;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class WavefrontNozzleApplicationTests {
+
+	@Test
+	public void contextLoads() {
+		// TODO ...
```

Comment:
Will approve the change when we have some kind of test (just testing whether the spring wirings can be satisfied would be a good start)